packages/language-server/src/completion-items.ts

```diff
@@ -118,11 +120,15 @@ export function computeLSPKind(
   }
 }
 
-function createTextEdit(
+function createTextEdits(
```

Comment:
This function is now ~120 lines, We keep trying to avoid it buy I think it is time to split it up...

Reply:
I extracted the handling for classes which was the most complex